Makefile

```diff
@@ -121,6 +121,35 @@ publish: prep-install-scripts
 	bin/porter mixins feed generate -d bin/mixins -f bin/atom.xml -t build/atom-template.xml
 	az storage blob upload -c porter -n atom.xml -f bin/atom.xml
 
+# all-bundles loops through all items under the dir provided by the first argument
+# and if the item is a sub-directory containing a porter.yaml file,
+# runs the make target(s) provided by the second argument
+define all-bundles
+	@for dir in $$(ls -1 $(1)); do \
+		if [[ -e "$(1)/$$dir/porter.yaml" ]]; then \
+			BUNDLE=$$dir make $(MAKE_OPTS) $(2) ; \
+		fi ; \
+	done
+endef
+
+EXAMPLES_DIR := examples
+
+.PHONY: build-bundle
+build-bundle:
+ifndef BUNDLE
```

Comment:
I'm no make wizard, seems like there are two ways to call this:

`make build-bundle` which will do them all and also `make build-bundle BUNDLE=wordpress`. Is that right?

Reply:
Yea!  Should I issue a follow-up PR mentioning this, maybe in a dev doc?  In case devs would like to push to their own registries?  (And actually, if we'd like to support pushing to non-deislabs registries, we'll have to add a `sed`-like string substitution on the inv image and bundle tag...)

Reply:
No that's fine, I was just asking for my own book learnin'. 😀